api/envoy/service/status/v3/csds.proto

```diff
@@ -88,7 +90,13 @@ message PerXdsConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.PerXdsConfig";
 
+  // Config status is generated by management servers.
   ConfigStatus status = 1;
+
+  // Client side config status is populated by xDS clients. Only one of the
+  // config status should present in one PerXdsConfig message. No matter what
+  // the client side config status is, xDS clients should always dump the most
+  // recent accepted xDS config.
   ClientSideConfigStatus client_side_config_status = 7;
```

Comment:
Suggest just calling this `client_status`.

Reply:
Name changed.